src/mudlet.cpp

```diff
@@ -4306,3 +4559,501 @@ QPair<bool, QStringList> mudlet::getLines(Host* pHost, const QString& windowName
         return qMakePair(false, failMessage);
     }
 }
+
+Hunhandle* mudlet::prepareProfileDictionary(const QString& hostName, QSet<QString>& wordSet)
```

Comment:
Not IMHO -  the dictionary file `.dic` (list of words as we need it) is likely to already exist if it has been used before - however this takes that file which ***can** have been **manually** edited by the user* - and rechecks the word count on the first line and recalculates the grapheme usages within so that the `TRY ` line in the `.aff` file can be revised to match the current contents of the `.dic` file - which is needed for `Hunspell_suggest` to work and to work in an optimum manner.

As it happens the existing `.aff` file is replaced here (and in the `saveDictionary` call).

Anyhow - this is about preparing the files for use by the Hunspell library - and it only **creates** the `.dic` on first usage so *I* think the current name is more appropriate.